<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(85, 255, 255)</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>591</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="login_page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>用户登录</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>401</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>224</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="username"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>280</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QPushButton" name="login">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/balckButton.png);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>240</y>
       <width>282</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>记住密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="notReg">
        <property name="styleSheet">
         <string notr="true">color: rgb(230, 50, 14);
text-decoration: underline;
font: 10pt &quot;黑体&quot;;</string>
        </property>
        <property name="text">
         <string>没有账号，立即注册</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="set_page">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>441</width>
       <height>351</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>285</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>服务器设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>84</x>
        <y>130</y>
        <width>90</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>服务器地址：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>84</x>
        <y>190</y>
        <width>90</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>服务器端口：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>130</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/balckButton.png);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="reg_page">
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>461</width>
       <height>361</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>260</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户注册</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/images/balckButton.png);</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>昵称</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>邮箱</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>确认密码</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="TitleWidget" name="title_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header location="global">titlewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
